src/display/api.js

```diff
@@ -201,6 +201,9 @@ if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("PRODUCTION")) {
  *   `OffscreenCanvas` in the worker. Primarily used to improve performance of
  *   image conversion/rendering.
  *   The default value is `true` in web environments and `false` in Node.js.
+ * @property {boolean} [canvasMaxAreaInBytes] - The integer value is used to
+ *   know when an image must be resized. If it's -1 then a possibly slow
+ *   algorithm is used to guess the max value.
```

Comment:
Let's briefly mention that this relies on OffscreenCanvas in the worker.
```suggestion
 * @property {boolean} [canvasMaxAreaInBytes] - The integer value is used to
 *   know when an image must be resized (uses `OffscreenCanvas` in the worker).
 *   If it's -1 then a possibly slow algorithm is used to guess the max value.
```